apps/frontend/pages/api/twitch/notification/[...param].ts

```diff
@@ -69,6 +72,26 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
     if (messageType === MessageType.Notification) {
         console.log(req.body.event);
+        const streamStatus = req.body.subscription.type;
+
+        // get the features that are enabled
+        const response = await axios.get(`${env.NEXTAUTH_URL}/api/feature/${param[0]}`);
```

Comment:
Extract `param[0]` as a constant like `const userId: string = param[0];` so it's more clear what param[0] is. Make sure to replace in all places you use it.